<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>C_FlaSetProgressWidget</class>
 <widget class="QWidget" name="C_FlaSetProgressWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>C_FlaSetProgressWidget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1,0">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="stw::opensyde_gui_elements::C_CamOgeWiSettingSubSection" name="pc_WiHeader" native="true"/>
   </item>
   <item>
    <widget class="QWidget" name="pc_WiContent" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>10</number>
      </property>
      <property name="topMargin">
       <number>9</number>
      </property>
      <property name="rightMargin">
       <number>10</number>
      </property>
      <property name="bottomMargin">
       <number>1</number>
      </property>
      <item>
       <widget class="stw::opensyde_gui_elements::C_FlaOgeTebProgressLog" name="pc_TextBrowser"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>9</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>stw::opensyde_gui_elements::C_CamOgeWiSettingSubSection</class>
   <extends>QWidget</extends>
   <header>C_CamOgeWiSettingSubSection.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>stw::opensyde_gui_elements::C_FlaOgeTebProgressLog</class>
   <extends>QTextBrowser</extends>
   <header>C_FlaOgeTebProgressLog.hpp</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
